admin/lint/checks.go

```diff
@@ -122,7 +122,7 @@ func checkGolint(workspaceDir string, file string) error {
 // checkAll runs all possible checks on a file.  Returns true if all checks pass, and false
 // otherwise.  Error details are dumped to stderr.
 func checkAll(workspaceDir string, file string) bool {
-	isBuildFile := filepath.Base(file) == "BUILD.bazel" || filepath.Ext(file) == ".bzl" || filepath.Base(file) == "Makefile.in"
+	isBuildFile := filepath.Base(file) == "Makefile.in"
```

Comment:
Just inline this below now. It's only used to skip tab checks, which is specific to Makefiles and not build files generally.

Reply:
I was about to do this... but realized that it's also used as part of the definition of `isDocumentation`. I think it makes sense to keep it.